ppml/trusted-big-data-ml/scala/docker-occlum/run_spark_on_occlum_glibc.sh

```diff
@@ -165,31 +176,71 @@ build_spark() {
     cp -rf /opt/spark-source image/opt/
 
     # Build
+    occlum build
+
+    #before start occlum app after occlum build
     if [[ $ATTESTATION == "true" ]]; then
-       occlum build --image-key /opt/occlum_spark/data/image_key
-       build_initfs
-    else
-       occlum build
+        if [[ $PCCS_URL == "" ]]; then
+            echo "[ERROR] Attestation set to true but NO PCCS"
+            exit 1
+        else
+            if [[ $RUNTIME_ENV == "driver" || $RUNTIME_ENV == "native" ]]; then
+                #verify ehsm service
```

Comment:
verify ehsm service is a bi-direction attestation. We need another env to enable this.